dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -350,7 +358,9 @@ const DesignableBanner: ReactComponent<BannerRenderProps> = ({
 									}}
 									tickerData={tickerSettings.tickerData}
 									tickerStylingSettings={
-										templateSettings.tickerStylingSettings
+										tickerSettings.name === 'AU'
```

Comment:
the banner colours are configured from the banner design tool, so this shouldn't be necessary